src/tasks/mixin/battle_mixin.py

```
@@ -94,8 +98,13 @@ def _register_battle_config(self):
         self.config_type.update(BATTLE_CONFIG_TYPE)
         self.config_type[BATTLE_CONFIG_MODE_KEY] = battle_mode_type
 
+    # 实时条件相关配置始终使用全局值（设计决策：仅全局配置）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)